spotbugs-exclude.xml

```diff
@@ -48,7 +48,7 @@
 
     <!-- Ignore GC call in CassandraTestTemplate as we want to GC after cluster shutdown -->
     <Match>
-        <Class name="org.apache.cassandra.sidecar.testing.CassandraTestTemplate" />
+        <Class name="org.apache.cassandra.sidecar.test.CassandraTestTemplate" />
```

Comment:
when using `testing` IntelliJ was not able to locate this class, which caused a lot of issues in IDE. So I decided to move the package from testing -> test. Now everything is good it seems